gips/datahandler/worker.py

```diff
@@ -226,3 +226,10 @@ def export_and_aggregate(job_id, start_ext, end_ext,
 
     if cleanup:
         shutil.rmtree(outdir)
+
+
+def look_busy(duration=5):
```

Comment:
It looks like this PR will need a commit or two before merging so I'll take it out while I'm in there.